src/Umbraco.Core/PropertyEditors/ValueConverters/EmailAddressValueConverter.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Umbraco.Core.Models.PublishedContent;
+
+namespace Umbraco.Core.PropertyEditors.ValueConverters
+{
+    public class EmailAddressValueConverter : PropertyValueConverterBase, IPropertyValueConverterMeta
```

Comment:
Purely so that Models Builder generates class properties of type string instead of object